tests/__init__.py

```diff
@@ -117,12 +117,10 @@ async def _create_db():
 
 
 class _LDAPServerFactory(ServerFactory):
-    protocol = LDAPServer
-
-    def __init__(self, root, ldap_server):
+    def __init__(self, root, ldap_server_type: Type[LDAPServer] = LDAPServer):
         self.root = root
-        if ldap_server:
-            self.protocol = ldap_server
+        if ldap_server_type:
```

Comment:
I think this condition needs to be removed.

Reply:
Done